code/frontend/test/SpeechToText.test.ts

```diff
@@ -1,16 +1,22 @@
-import { expect } from "chai";
-import { it } from "mocha";
+import { beforeEach, describe, expect, it } from 'vitest'
 import { multiLingualSpeechRecognizer } from "../src/util/SpeechToText.js";
 
+const recognizedLanguages = "en-US,es-ES";
+
 describe("SpeechToText", () => {
+  beforeEach(() => {
+    import.meta.env.VITE_SPEECH_RECOGNIZER_LANGUAGES = recognizedLanguages;
+  })
+
   it("creates a speech recognizer with multiple languages", async () => {
     const key = "key";
     const region = "region";
-    const languages = ["en-US", "fr-FR", "de-DE", "it-IT"];
-    const recognizer = multiLingualSpeechRecognizer(key, region, languages);
+
+
+    const recognizer = multiLingualSpeechRecognizer(key, region);
 
     expect(recognizer.properties.getProperty("SpeechServiceConnection_Key")).to.equal(key);
     expect(recognizer.properties.getProperty("SpeechServiceConnection_Region")).to.equal(region);
-    expect(recognizer.properties.getProperty("SpeechServiceConnection_AutoDetectSourceLanguages")).to.equal(languages.join(","));
+    expect(recognizer.properties.getProperty("SpeechServiceConnection_AutoDetectSourceLanguages")).to.equal(recognizedLanguages);
   });
 });
```

Comment:
Is it worth adding a test for the defaulting of the languages if the env var is not set?